modules/op-release-notes-1-8.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-8_{context}"]
+= Release notes for {pipelines-title} General Availability 1.8
+
+With this update, {pipelines-title} General Availability (GA) 1.8 is available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="new-features-1-8_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
+
+[id="<feature>-new-features-1-8_{context}"]
+=== <feature>
+
+*
+
+
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+* Support for the `operator.tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release. Bug fixes and support for that API version are provided only through the end of the current GA 1.8 lifecycle. Use the `v1beta1` API version instead.
```

Comment:
This is `tekton.dev/v1alpha1` that is deprecated. This solely affect the pipeline component (`TaskRun`, `PipelineRun`, `Task`, `Pipeline`, …)

`operator.tekton.dev/v1alpha1` is not deprecated and still in use by the operator.

Reply:
Updated, thanks. PTAL, @vdemeester. WDYT?

Reply:
sounds good 👍🏼